src/status_im/contexts/wallet/send/select_collectible_amount/view.cljs

```diff
@@ -46,24 +43,23 @@
        :container-style style/network-tags-container}]
 
      [quo/amount-input
-      {:max-value       (if (integer? balance) balance 0)
-       :min-value       1
-       :value           value
-       :on-inc-press    inc-value
-       :on-dec-press    dec-value
+      {:max-value       (controlled-input/upper-limit value)
+       :min-value       (controlled-input/lower-limit value)
+       :value           (controlled-input/numeric-value value)
+       :on-inc-press    (fn [] (set-value #(controlled-input/increase %)))
+       :on-dec-press    (fn [] (set-value #(controlled-input/decrease %)))
```

Comment:
It'd be great if we provide stable references to these callbacks, to improve potential performance issues due to re-renders.

BTW, we can directly pass the function to `set-value` no need of a wrapper:

```clojure
(set-value #(controlled-input/decrease %))
;; ->
(set-value controlled-input/decrease)
```